flow/connectors/postgres/validate.go

```diff
@@ -25,10 +26,26 @@ func (c *PostgresConnector) CheckSourceTables(ctx context.Context,
 		var row pgx.Row
 		tableArr = append(tableArr, fmt.Sprintf(`(%s::text,%s::text)`,
 			QuoteLiteral(parsedTable.Schema), QuoteLiteral(parsedTable.Table)))
+
+		selectedColumnsStr := "*"
+		if _, ok := excludedColumnsMap[parsedTable.Schema+"."+parsedTable.Table]; ok {
```

Comment:
This is always going to be a pain, since we parsed `req.ConnectionConfigs.TableMappings` for `tableNames` while `excludedColumnsMap` is built on the raw strings from `req.ConnectionConfigs.TableMappings`. So you can't really use `parsedTable.String()` but that's what you'd want if you were going to properly handle cases like `"sch.e""ma"."ta.b""le"` which parses to `sch.e"ma.ta.b"le`